go.sum

```diff
@@ -1,4 +1,12 @@
 cloud.google.com/go v0.26.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
+cloud.google.com/go v0.31.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
```

Comment:
A go mod tidy perhaps? Or completely remove `go.sum`, then do a `go build`?

Reply:
done

Reply:
both of them. `go mod tidy` was already run before your comment so no change, so I removed `go.sum` and then I run a `go build`